setup.py

```diff
@@ -46,7 +44,7 @@
 with open(fname) as f:
     tests_require = [l.strip() for l in f.readlines()]
 
-if sys.version_info[:2] < (3, 3):
+if sys.version_info[0] < 3:
```

Comment:
Since we are going to drop the if statement soon (when we drop the Python 2 support), I'd prefer leaving this as-is.

Reply:
Also, please update the following line in `setup.py`:

```py
python_requires='>=2.6, !=3.0.*, !=3.1.*',
```

Reply:
✅